src/coreclr/nativeaot/Runtime/MathHelpers.cpp

```diff
@@ -56,25 +56,25 @@ FCIMPL1_D(uint32_t, RhpDbl2UInt, double val)
 FCIMPLEND
 
 #ifndef HOST_64BIT
-EXTERN_C int64_t QCALLTYPE RhpLDiv(int64_t i, int64_t j)
+EXTERN_C int64_t QCALLTYPE InternalDivInt64(int64_t i, int64_t j)
```

Comment:
QCALLTYPE won't work together with `[MethodImpl(MethodImplOptions.InternalCall)]` on the managed side